service/src/main/resources/application.yml

```diff
@@ -89,6 +89,12 @@ spring:
       hibernate:
         format_sql: true
 
+  config:
+    import:
+      - optional:classpath:pipelines-config.yml
+      - optional:file:../config/local-properties.yml
```

Comment:
interesting. what does this and the next line do?

Reply:
great eyes, this actually wasnt needed and i had deleted it after i asked you to look at this code 😬